packages/eslint-plugin-gestalt/src/__fixtures__/prefer-box-as-tag/valid.js

```diff
@@ -0,0 +1,9 @@
+import { Box } from 'gestalt';
+export default function TestElement() {
+  return (
+    <Box>
+      <ul />
+      <frame />
```

Comment:
Good old times, so happy we no longer have to use `<frame />` in today's web development: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frame